sources/Application/Instruments/MidiInstrument.cpp

```diff
@@ -139,6 +141,51 @@ bool MidiInstrument::Render(int channel, fixed *buffer, int size,
 
     first_[channel] = false;
   }
+
+  // Update pitch bend.
+  if (updateTick) {
+    if (pitchBend_) {
+      int prev = pitchBendCurrent_;
+      if (pitchBendSpeed_ == 0) {
+        pitchBendCurrent_ = pitchBendTarget_;
+      } else {
+        int diff = pitchBendTarget_ - pitchBendCurrent_;
+        int sign = (diff > 0) ? 1 : -1;
+        int nextValue =
```

Comment:
please use specific size ints. I know the legacy lgpt code doesn't but we are trying to fix that in all new code.